src/app/components/mnoe-users-local-list/mnoe-users-local-list.html

```diff
@@ -32,9 +35,16 @@
               <span ng-show="user.isSendingInvite"><i class="fa fa-spinner fa-pulse fa-fw"></i></span>
               {{'mnoe_admin_panel.dashboard.users.widget.local_list.resend_invite' | translate}}
             </button>
+            <button ng-show="user.status === 'invited' || user.status === 'pending'" class="btn btn-xs btn-warning" ng-disabled="user.isSendingInvite" ng-click="removeMember(user)">
+              {{'mnoe_admin_panel.dashboard.users.widget.local_list.cancel_invite' | translate}}
+            </button>
             <span ng-show="user.status === 'active'" translate>mnoe_admin_panel.dashboard.users.widget.local_list.active</span>
           </td>
           <td>
+            <select ng-init="user.role=availableRoles()[indexOfRoleInTable(user.role)]" ng-model="user.role" ng-options="role | translate for role in availableRoles()" ng-change="updateUserRole(user)"></select>
```

Comment:
`ng-change="updateUserRole(user)"` Do you need the full object? why not pass only id or email?

Can extract `user.role=availableRoles()[indexOfRoleInTable(user.role)]` to a function?

Reply:
I need to access the role in the method updateUserRole, and I need the email in the mnoe request, so it's easier to pass the whole object rather than two separate arguments related to the same model